owncloudApp/src/main/java/com/owncloud/android/extensions/ActivityExt.kt

```diff
@@ -334,6 +341,33 @@ private fun Activity.showSelectSecurityDialog(
     }
 }
 
+fun Activity.feedbackBehaviour(feedbackMail: String) {
+    if (feedbackMail.isNotEmpty()) {
+        val feedbackMail = getString(R.string.mail_feedback)
```

Comment:
Why are you retrieving `feedbackMail` again when you already have it as parameter?

Reply:
My mistake, removed.